operators/README.md

```diff
@@ -113,6 +113,19 @@ If the request for a new snapshot is valid, a new Job is created that performs t
 
 When the snapshot creation process successfully terminates, the docker registry will contain a new VM image with the exact copy of the target persistent VM at the moment of the snapshot creation. Note that before being able to create a new VM instance with that image, you should first create a new Template with the newly uploaded image.
 
+### Personal storage
+
+The Instance Operator, can attach to the instance the user's personal storage using the parameters stored in the `mydrive-info` secret created by the Tenant Operator. 
+
+A flag named `MountMyDriveVolume` in the `Environment` inside the `Template` of the `Instance` resource to reconcile is used to decide whether to mount or not the personal storage.
```

Comment:
I cannot understand the sentence. What does it mean "to reconcile" here?

Reply:
The operation that the Instance Operator does where it changes the Instance status from the actual state to the desired state is called reconciliation.

Reply:
I know what "reconciliation" is :-) , but the sentence looks weird to me... can you please rephrase to make it more clear? I cannot do myself, as I don't understand what you wanted to say :-(

Reply:
I was trying to say that the flag is found inside the Environment that is inside the Instance resource that is being reconciled by the operator.
I could write the path to the flag like: `instance.environments\[\].MountMyDriveVolume` but I don't know if that is clear enough.
Another way could be:
```
With the `MountMyDriveVolume` flag inside each `Environment` found in a Instance Resource the Instance Operator mounts or not the user's personal storage inside that `Environment`
```

Reply:
That last version should be more clear

Reply:
I made a sentence based on @matteorosani suggestion